src/npctalk.cpp

```diff
@@ -2693,6 +2697,23 @@ void conditional_t::set_is_driving( bool is_npc )
     };
 }
 
+void conditional_t::set_u_has_stolen_item()
```

Comment:
```suggestion
void conditional_t::set_has_stolen_item( bool is_npc )
```
please be consistent with the other conditionals.